operations/enterprises.md

```diff
@@ -726,6 +726,17 @@ Adds a new company to the enterprise.
 | `BillingCode` | string | optional | Billing code of the company. |
 | `AccountingCode` | string | optional | Accounting code of the company. |
 | `Address` | [Address](configuration.md#address) | optional | Address of the company. |
+| `Options` | Array of [Company option](enterprises.md#company-options)  | optional | Options of the company. |
+| `InviceDueInterval` | string | optional | Start of the interval in UTC timezone in ISO 8601 format. |
```

Comment:
InvoiceInterval is not start of the interval

Reply:
So it's timeframe whe the invoice ca be paid with the latest day being the end of payment terms? 
`Interval in UTC timezone in ISO 8601 format.`